### ISSUE TITLE
0095.bag is deleted

### ISSUE THREAD

**Issue by Ciao-li** [LeisureLei/VLO#1]
Hello, 0095.bag seems to have been deleted, can you send another copy?

**Comment by LeisureLei** [comment]
Yes, baidudisk delete all ! I lose data, too. Maybe you can download from KITTI Raw data http://www.cvlibs.net/datasets/kitti/raw_data.php.

**Comment by Ciao-li** [comment]
@LeisureLei
Hello, I downloaded a small ros package, which is available here.
https://pan.baidu.com/s/11tp26S5dy4t8wU85ZSf-tA 
jvtb
By the way, if I want to display the results, do I need to modify the code?

**Comment by LeisureLei** [comment]
> @LeisureLei
> Hello, I downloaded a small ros package, which is available here.
> https://pan.baidu.com/s/11tp26S5dy4t8wU85ZSf-tA
> jvtb
> By the way, if I want to display the results, do I need to modify the code?

You may need to change the topic names subscribed in lidtocam.cpp and laserhandle.cpp and extrinsic parameters between lidar and camera in transformProject function of lidtofcam.h.

**Comment by Ciao-li** [comment]
> > @LeisureLei
> > Hello, I downloaded a small ros package, which is available here.
> > https://pan.baidu.com/s/11tp26S5dy4t8wU85ZSf-tA
> > jvtb
> > By the way, if I want to display the results, do I need to modify the code?
> 
> You may need to change the topic names subscribed in lidtocam.cpp and laserhandle.cpp and extrinsic parameters between lidar and camera in transformProject function of lidtofcam.h.

Ok i got it, thank you very much for your reply
